File: test/javascript/negatives/F1.js
function processData(inputList) {
    return helper(inputList);
}

function helper(lst) {
    let total = 0;
    for (const x of lst) {
        if (Array.isArray(x)) {
            total += helper(x);
        } else if (typeof x === "number") {
            total += x;
        }
    }
    return total;
}

module.exports = { processData };

File: test/java/negatives/F1.java
import java.util.List;

public class F1 {
    public static int processData(List<Object> inputList) {
        return helper(inputList);
    }

    private static int helper(List<Object> lst) {
        int total = 0;
        for (Object x : lst) {
            if (x instanceof List) {
                total += helper((List<Object>) x);
            } else if (x instanceof Integer) {
                total += (Integer) x;
            }
        }
        return total;
    }
}